l10n_be_cooperator/report/tax_shelter_report_templates.xml

```diff
@@ -198,7 +201,10 @@ SPDX-License-Identifier: AGPL-3.0-or-later
                     <p>
                         Pour bénéficier de la réduction d'impôts, vous pouvez joindre
                         cette lettre à votre déclaration fiscale pour les revenus
-                        <span t-field="o.declaration_id.fiscal_year" />.
+                        <span
+                            t-field="o.declaration_id.fiscal_year"
+                            t-options-widget="chars"
```

Comment:
why sometimes `"'chars'"` and sometimes `"chars"`?

Reply:
Should the widget not be `char` in any case?

Reply:
yes, also t-options-widget seems to be deprecated, I used t-options instead.
I tested it and it works.